python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -948,7 +997,64 @@ def reset_index(
             self.index = cudf_series.index
             return None
 
-    def contains_properly(self, other, align=False, allpairs=False):
+    def contains(self, other, align=False, allpairs=False, mode="full"):
+        """Returns a `Series` of `dtype('bool')` with value `True` for each
+        aligned geometry that contains _other_.
+
+        An object `a` is said to contain `b` if `b`'s `boundary` and
+        `interiors` are within those of `a` and no point of `b` lies in the
+        exterior of `a`.
```

Comment:
Let's avoid backtick abuse. It should only be reserved for code.

Reply:
done